src/components/CardWithIcon/CardWithIcon.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
```

Comment:
i see mixed casing and indents, i guess we dont have the linter set up properly for tsx files. no need to deal with that manually.